Game crashes when a gemstone is mined

### Firmament Version

44.0.0 for 1.21.11

### Bug Description

The game crashes when mining gemstones. Through some testing, I found that it doesn't happen every time I mined a gemstone, but it happens very consistently. I turned off all my resource packs and disabled all other mods (except for Firmament's mandatory dependencies) and the crash still occurs. In the session where I created the logs, it happened on the sixth gemstone I mined.

[crash-2026-02-27_11.13.15-client.txt](https://github.com/user-attachments/files/25608666/crash-2026-02-27_11.13.15-client.txt)

### Minecraft Log

```plain text
[11:12:03] [main/INFO]: Loading Minecraft 1.21.11 with Fabric Loader 0.18.4
[11:12:03] [main/INFO]: Loading 68 mods:
	- fabric-api 0.141.3+1.21.11
	   |-- fabric-api-base 1.0.5+4ebb5c083e
	   |-- fabric-api-lookup-api-v1 1.6.114+20dc27073e
	   |-- fabric-biome-api-v1 17.1.1+4fc5413f3e
	   |-- fabric-block-api-v1 1.1.10+4ebb5c083e
	   |-- fabric-block-view-api-v2 1.0.39+4ebb5c083e
	   |-- fabric-command-api-v2 2.4.7+6b42a6003e
	   |-- fabric-content-registries-v0 10.2.14+4fc5413f3e
	   |-- fabric-convention-tags-v1 2.1.55+7f945d5b3e
	   |-- fabric-convention-tags-v2 2.17.3+8ef948ba3e
	   |-- fabric-crash-report-info-v1 0.3.23+4ebb5c083e
	   |-- fabric-data-attachment-api-v1 1.8.47+eed0806f3e
	   |-- fabric-data-generation-api-v1 23.4.0+69974c4e3e
	   |-- fabric-dimensions-v1 4.0.28+4fc5413f3e
	   |-- fabric-entity-events-v1 3.1.1+1d0ab4303e
	   |-- fabric-events-interaction-v0 4.1.0+3b89ecf63e
	   |-- fabric-game-rule-api-v1 2.0.3+4fc5413f3e
	   |-- fabric-item-api-v1 11.5.20+d0c46b9e3e
	   |-- fabric-item-group-api-v1 4.2.36+4fc5413f3e
	   |-- fabric-key-binding-api-v1 1.1.7+4fc5413f3e
	   |-- fabric-lifecycle-events-v1 2.6.15+4ebb5c083e
	   |-- fabric-loot-api-v2 3.0.73+3f89f5a53e
	   |-- fabric-loot-api-v3 2.0.20+78c8b4663e
	   |-- fabric-message-api-v1 6.1.12+4ebb5c083e
	   |-- fabric-model-loading-api-v1 6.0.15+4fc5413f3e
	   |-- fabric-networking-api-v1 5.1.5+ae1e07683e
	   |-- fabric-object-builder-api-v1 21.1.39+4fc5413f3e
	   |-- fabric-particles-v1 4.2.11+4fc5413f3e
	   |-- fabric-recipe-api-v1 8.2.3+4ebb5c083e
	   |-- fabric-registry-sync-v0 6.2.5+1718722b3e
	   |-- fabric-renderer-api-v1 8.0.3+f4ffd2e53e
	   |-- fabric-renderer-indigo 5.0.3+f4ffd2e53e
	   |-- fabric-rendering-fluids-v1 3.1.43+4ebb5c083e
	   |-- fabric-rendering-v1 16.2.10+0290ad933e
	   |-- fabric-resource-conditions-api-v1 5.0.35+4fc5413f3e
	   |-- fabric-resource-loader-v0 3.3.4+4fc5413f3e
	   |-- fabric-resource-loader-v1 1.0.10+78c8b4663e
	   |-- fabric-screen-api-v1 3.1.7+4ebb5c083e
	   |-- fabric-screen-handler-api-v1 1.3.161+4fc5413f3e
	   |-- fabric-serialization-api-v1 1.0.5+4ebb5c083e
	   |-- fabric-sound-api-v1 1.0.51+4fc5413f3e
	   |-- fabric-tag-api-v1 1.2.20+4fc5413f3e
	   |-- fabric-transfer-api-v1 6.0.25+4fc5413f3e
	   \-- fabric-transitive-access-wideners-v1 7.1.0+014c8cec3e
	- fabric-language-kotlin 1.13.9+kotlin.2.3.10
	   |-- org_jetbrains_kotlin_kotlin-reflect 2.3.10
	   |-- org_jetbrains_kotlin_kotlin-stdlib 2.3.10
	   |-- org_jetbrains_kotlin_kotlin-stdlib-jdk7 2.3.10
	   |-- org_jetbrains_kotlin_kotlin-stdlib-jdk8 2.3.10
	   |-- org_jetbrains_kotlinx_atomicfu-jvm 0.31.0
	   |-- org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm 1.10.2
	   |-- org_jetbrains_kotlinx_kotlinx-coroutines-jdk8 1.10.2
	   |-- org_jetbrains_kotlinx_kotlinx-datetime-jvm 0.7.1
	   |-- org_jetbrains_kotlinx_kotlinx-io-bytestring-jvm 0.8.2
	   |-- org_jetbrains_kotlinx_kotlinx-io-core-jvm 0.8.2
	   |-- org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm 1.10.0
	   |-- org_jetbrains_kotlinx_kotlinx-serialization-core-jvm 1.10.0
	   \-- org_jetbrains_kotlinx_kotlinx-serialization-json-jvm 1.10.0
	- fabricloader 0.18.4
	- firmament 44.0.0+mc1.21.11
	   |-- cloth-basic-math 0.6.1
	   |-- hypixel-mod-api 1.0.1+build.1+mc1.21
	   |    \-- net_hypixel_mod-api 1.0.1
	   |-- jarviscci 2.1.0
	   |-- mixinextras 0.5.3
	   \-- mm_shedaniel 2.4.1
	- java 21
	- minecraft 1.21.11
[11:12:03] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.7 Source=file:/home/netherite/.local/share/PrismLauncher/libraries/net/fabricmc/sponge-mixin/0.17.0+mixin.0.8.7/sponge-mixin-0.17.0+mixin.0.8.7.jar Service=Knot/Fabric Env=CLIENT
[11:12:03] [main/INFO]: Compatibility level set to JAVA_21
[11:12:04] [main/WARN]: Reference map 'Firmament-refmap.json' for firmament.mixins.json could not be read. If this is a development environment you can ignore this message
[11:12:04] [main/INFO]: Started MM (shedaniel's fork) at me/shedaniel/gen/mixin/
[11:12:04] [main/WARN]: Reference map 'Manningham Mills-refmap.json' for mixins.mm_shedaniel.json could not be read. If this is a development environment you can ignore this message
[11:12:05] [main/INFO]: Initializing MixinExtras via com.llamalad7.mixinextras.service.MixinExtrasServiceImpl(version=0.5.3).
[11:12:06] [Datafixer Bootstrap/INFO]: 287 Datafixer optimizations took 537 milliseconds
[11:12:10] [Render thread/INFO]: Environment: Environment[sessionHost=https://sessionserver.mojang.com, servicesHost=https://api.minecraftservices.com, profilesHost=https://api.mojang.com, name=PROD]
[11:12:10] [Render thread/INFO]: Setting user: NetheriteMiner
[11:12:10] [Render thread/INFO]: Definitely not up to no good
[11:12:10] [Render thread/INFO]: [Indigo] Registering Indigo renderer!
[11:12:12] [DefaultDispatcher-worker-2/INFO]: Updating collection data
[11:12:12] [DefaultDispatcher-worker-3/INFO]: Updating NEU prices
[11:12:12] [DefaultDispatcher-worker-4/INFO]: Updating bazaar prices
[11:12:12] [Render thread/INFO]: Registered clientbound packet with identifier 'hypixel:hello'
[11:12:12] [Render thread/INFO]: Registered clientbound packet with identifier 'hypixel:player_info'
[11:12:12] [Render thread/INFO]: Registered clientbound packet with identifier 'hyevent:location'
[11:12:12] [Render thread/INFO]: Registered clientbound packet with identifier 'hypixel:party_info'
[11:12:12] [Render thread/INFO]: Registered clientbound packet with identifier 'hypixel:ping'
[11:12:12] [Render thread/INFO]: Registered serverbound packet with identifier 'hypixel:player_info'
[11:12:12] [Render thread/INFO]: Registered serverbound packet with identifier 'hypixel:party_info'
[11:12:12] [Render thread/INFO]: Registered serverbound packet with identifier 'hypixel:ping'
[11:12:12] [Render thread/INFO]: Registered serverbound packet with identifier 'hypixel:register'
[11:12:12] [Render thread/INFO]: Registering mod resource pack
[11:12:12] [DefaultDispatcher-worker-4/INFO]: Downloaded 86 collections
[11:12:12] [DefaultDispatcher-worker-4/INFO]: Planning to upgrade repository from 0f8cf73e1a19486968d789f01e6f1bbb514774f0 to 2bd18209659fb4d039aeebdd3f790da9b4566ef8 from https://github.com/NotEnoughUpdates/NotEnoughUpdates-REPO/archive/2bd18209659fb4d039aeebdd3f790da9b4566ef8.zip
[11:12:12] [Render thread/INFO]: Registering mod resource pack
[11:12:12] [Render thread/INFO]: Registering mod resource pack
[11:12:12] [Render thread/INFO]: Backend library: LWJGL version 3.3.3-snapshot
[11:12:13] [DefaultDispatcher-worker-4/INFO]: Download repository zip file to /tmp/firmament-repo8244776873549616035.zip. Deleting old repository
[11:12:13] [DefaultDispatcher-worker-4/INFO]: Extracting new repository
[11:12:13] [Render thread/INFO]: Using optional rendering extensions: GL_ARB_buffer_storage, GL_KHR_debug, GL_ARB_vertex_attrib_binding, GL_ARB_direct_state_access, GL_EXT_texture_filter_anisotropic
[11:12:13] [Render thread/INFO]: Registering mod resource pack
[11:12:13] [Render thread/INFO]: Registering mod resource pack
[11:12:13] [Render thread/INFO]: Registering mod resource pack
[11:12:13] [Render thread/WARN]: Removed resource pack continuity:glass_pane_culling_fix from options because it doesn't seem to exist anymore
[11:12:13] [Render thread/WARN]: Removed resource pack continuity:default from options because it doesn't seem to exist anymore
[11:12:14] [Render thread/INFO]: Reloading ResourceManager: vanilla, fabric-api, fabric-api-base, fabric-api-lookup-api-v1, fabric-biome-api-v1, fabric-block-api-v1, fabric-block-view-api-v2, fabric-command-api-v2, fabric-content-registries-v0, fabric-convention-tags-v1, fabric-convention-tags-v2, fabric-crash-report-info-v1, fabric-data-attachment-api-v1, fabric-data-generation-api-v1, fabric-dimensions-v1, fabric-entity-events-v1, fabric-events-interaction-v0, fabric-game-rule-api-v1, fabric-item-api-v1, fabric-item-group-api-v1, fabric-key-binding-api-v1, fabric-language-kotlin, fabric-lifecycle-events-v1, fabric-loot-api-v2, fabric-loot-api-v3, fabric-message-api-v1, fabric-model-loading-api-v1, fabric-networking-api-v1, fabric-object-builder-api-v1, fabric-particles-v1, fabric-recipe-api-v1, fabric-registry-sync-v0, fabric-renderer-api-v1, fabric-renderer-indigo, fabric-rendering-fluids-v1, fabric-rendering-v1, fabric-resource-conditions-api-v1, fabric-resource-loader-v0, fabric-resource-loader-v1, fabric-screen-api-v1, fabric-screen-handler-api-v1, fabric-serialization-api-v1, fabric-sound-api-v1, fabric-tag-api-v1, fabric-transfer-api-v1, fabric-transitive-access-wideners-v1, fabricloader, firmament, hypixel-mod-api, jarviscci, mm_shedaniel, programmer_art, fabric-item-group-api-v1_programmer_art, file/PACP_Base_Release, file/Opaque Glass
[11:12:14] [Worker-Main-2/INFO]: Loaded 0 global item overrides
[11:12:14] [Worker-Main-11/ERROR]: Invalid path in pack: minecraft:textures/block/black_stained_glass (copy).png, ignoring
[11:12:14] [DefaultDispatcher-worker-4/INFO]: Repository loaded on disk.
[11:12:14] [DefaultDispatcher-worker-4/INFO]: Repo reload started.
[11:12:14] [Worker-Main-7/WARN]: Invalid sounds.json in resourcepack: 'file/PACP_Base_Release'
com.google.gson.JsonParseException: com.google.gson.stream.MalformedJsonException: Expected value at line 1 column 3 path $
See https://github.com/google/gson/blob/main/Troubleshooting.md#malformed-json
	at knot//net.minecraft.class_3518.method_47992(class_3518.java:532)
	at knot//net.minecraft.class_3518.method_15297(class_3518.java:537)
	at knot//net.minecraft.class_1144.method_18180(class_1144.java:80)
	at knot//net.minecraft.class_1144.method_18789(class_1144.java:41)
	at knot//net.minecraft.class_4080.method_18791(class_4080.java:13)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at knot//net.minecraft.class_4014.method_67576(class_4014.java:51)
	at java.base/java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1423)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:387)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1312)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1843)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1808)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:188)
Caused by: com.google.gson.stream.MalformedJsonException: Expected value at line 1 column 3 path $
See https://github.com/google/gson/blob/main/Troubleshooting.md#malformed-json
	at knot//com.google.gson.stream.JsonReader.syntaxError(JsonReader.java:1817)
	at knot//com.google.gson.stream.JsonReader.doPeek(JsonReader.java:726)
	at knot//com.google.gson.stream.JsonReader.peek(JsonReader.java:547)
	at knot//com.google.gson.internal.bind.MapTypeAdapterFactory$Adapter.read(MapTypeAdapterFactory.java:177)
	at knot//com.google.gson.internal.bind.MapTypeAdapterFactory$Adapter.read(MapTypeAdapterFactory.java:161)
	at knot//net.minecraft.class_3518.method_47992(class_3518.java:530)
	... 12 more
[11:12:14] [Worker-Main-3/INFO]: Found unifont_all_no_pua-17.0.01.hex, loading
[11:12:15] [Worker-Main-9/INFO]: Found unifont_pua-17.0.01.hex, loading
[11:12:15] [Worker-Main-6/WARN]: Exception loading blockstate definition: 'minecraft:black_stained_glass_pane/file/Opaque Glass' for variant: 'inventory': Unknown blockstate property: 'inventory'
[11:12:15] [Worker-Main-6/WARN]: Exception loading blockstate definition: 'minecraft:black_stained_glass_pane/file/Opaque Glass' for variant: 'facing=north': Unknown blockstate property: 'facing'
[11:12:15] [Worker-Main-6/WARN]: Exception loading blockstate definition: 'minecraft:black_stained_glass_pane/file/Opaque Glass' for variant: 'facing=east': Unknown blockstate property: 'facing'
[11:12:15] [Worker-Main-6/WARN]: Exception loading blockstate definition: 'minecraft:black_stained_glass_pane/file/Opaque Glass' for variant: 'facing=south': Unknown blockstate property: 'facing'
[11:12:15] [Worker-Main-6/WARN]: Exception loading blockstate definition: 'minecraft:black_stained_glass_pane/file/Opaque Glass' for variant: 'facing=west': Unknown blockstate property: 'facing'
[11:12:15] [Worker-Main-3/INFO]: Found unifont_jp_patch-17.0.01.hex, loading
[11:12:17] [Render thread/INFO]: OpenAL initialized on device Built-in Audio Analog Stereo
[11:12:17] [Render thread/INFO]: Sound engine started
[11:12:17] [Render thread/INFO]: Created: 512x256x0 minecraft:textures/atlas/particles.png-atlas
[11:12:17] [Render thread/INFO]: Created: 128x128x0 minecraft:textures/atlas/decorated_pot.png-atlas
[11:12:17] [Render thread/INFO]: Created: 2048x1024x0 minecraft:textures/atlas/armor_trims.png-atlas
[11:12:17] [Render thread/INFO]: Created: 512x256x0 minecraft:textures/atlas/paintings.png-atlas
[11:12:17] [Render thread/INFO]: Created: 1024x512x0 minecraft:textures/atlas/shield_patterns.png-atlas
[11:12:17] [Render thread/INFO]: Created: 1024x512x0 minecraft:textures/atlas/blocks.png-atlas
[11:12:18] [Render thread/INFO]: Created: 512x512x0 minecraft:textures/atlas/chest.png-atlas
[11:12:18] [Render thread/INFO]: Created: 256x128x0 minecraft:textures/atlas/celestials.png-atlas
[11:12:18] [Render thread/INFO]: Created: 1024x512x0 minecraft:textures/atlas/banner_patterns.png-atlas
[11:12:18] [Render thread/INFO]: Created: 512x512x0 minecraft:textures/atlas/beds.png-atlas
[11:12:18] [DefaultDispatcher-worker-4/INFO]: Repo reload completed.
[11:12:18] [Render thread/INFO]: Created: 1024x512x0 minecraft:textures/atlas/items.png-atlas
[11:12:18] [Render thread/INFO]: Created: 1024x1024x0 minecraft:textures/atlas/gui.png-atlas
[11:12:18] [Render thread/INFO]: Created: 128x64x0 minecraft:textures/atlas/map_decorations.png-atlas
[11:12:18] [Render thread/INFO]: Created: 512x256x0 minecraft:textures/atlas/signs.png-atlas
[11:12:18] [Render thread/INFO]: Created: 512x512x0 minecraft:textures/atlas/shulker_boxes.png-atlas
[11:12:18] [Render thread/WARN]: Failed to issue a ClientboundUpdateRecipesPacket (to reload REI). This may lead to an outdated item list.
[11:12:21] [Render thread/ERROR]: ########## GL ERROR ##########
[11:12:21] [Render thread/ERROR]: @ Render
[11:12:21] [Render thread/ERROR]: 65547: X11: Standard cursor shape unavailable
[11:12:23] [Render thread/INFO]: Registering mod resource pack
[11:12:23] [Render thread/INFO]: Registering mod resource pack
[11:12:23] [Render thread/INFO]: Registering mod resource pack
[11:12:27] [Render thread/INFO]: Registering mod resource pack
[11:12:27] [Render thread/INFO]: Registering mod resource pack
[11:12:27] [Render thread/INFO]: Registering mod resource pack
[11:12:27] [Render thread/INFO]: Reloading ResourceManager: vanilla, fabric-api, fabric-api-base, fabric-api-lookup-api-v1, fabric-biome-api-v1, fabric-block-api-v1, fabric-block-view-api-v2, fabric-command-api-v2, fabric-content-registries-v0, fabric-convention-tags-v1, fabric-convention-tags-v2, fabric-crash-report-info-v1, fabric-data-attachment-api-v1, fabric-data-generation-api-v1, fabric-dimensions-v1, fabric-entity-events-v1, fabric-events-interaction-v0, fabric-game-rule-api-v1, fabric-item-api-v1, fabric-item-group-api-v1, fabric-key-binding-api-v1, fabric-language-kotlin, fabric-lifecycle-events-v1, fabric-loot-api-v2, fabric-loot-api-v3, fabric-message-api-v1, fabric-model-loading-api-v1, fabric-networking-api-v1, fabric-object-builder-api-v1, fabric-particles-v1, fabric-recipe-api-v1, fabric-registry-sync-v0, fabric-renderer-api-v1, fabric-renderer-indigo, fabric-rendering-fluids-v1, fabric-rendering-v1, fabric-resource-conditions-api-v1, fabric-resource-loader-v0, fabric-resource-loader-v1, fabric-screen-api-v1, fabric-screen-handler-api-v1, fabric-serialization-api-v1, fabric-sound-api-v1, fabric-tag-api-v1, fabric-transfer-api-v1, fabric-transitive-access-wideners-v1, fabricloader, firmament, hypixel-mod-api, jarviscci, mm_shedaniel
[11:12:27] [Worker-Main-7/INFO]: Loaded 0 global item overrides
[11:12:27] [Worker-Main-10/INFO]: Found unifont_all_no_pua-17.0.01.hex, loading
[11:12:27] [Worker-Main-3/INFO]: Found unifont_pua-17.0.01.hex, loading
[11:12:27] [Worker-Main-6/INFO]: Found unifont_jp_patch-17.0.01.hex, loading
[11:12:28] [Render thread/INFO]: OpenAL initialized on device Built-in Audio Analog Stereo
[11:12:28] [Render thread/INFO]: Sound engine started
[11:12:28] [Render thread/INFO]: Created: 512x256x0 minecraft:textures/atlas/particles.png-atlas
[11:12:28] [Render thread/INFO]: Created: 128x128x0 minecraft:textures/atlas/decorated_pot.png-atlas
[11:12:28] [Render thread/INFO]: Created: 2048x1024x0 minecraft:textures/atlas/armor_trims.png-atlas
[11:12:28] [Render thread/INFO]: Created: 512x256x0 minecraft:textures/atlas/paintings.png-atlas
[11:12:28] [Render thread/INFO]: Created: 1024x512x0 minecraft:textures/atlas/shield_patterns.png-atlas
[11:12:28] [Render thread/INFO]: Created: 1024x512x0 minecraft:textures/atlas/blocks.png-atlas
[11:12:28] [Render thread/INFO]: Created: 512x512x0 minecraft:textures/atlas/chest.png-atlas
[11:12:28] [Render thread/INFO]: Created: 256x128x0 minecraft:textures/atlas/celestials.png-atlas
[11:12:28] [Render thread/INFO]: Created: 1024x512x0 minecraft:textures/atlas/banner_patterns.png-atlas
[11:12:28] [Render thread/INFO]: Created: 512x512x0 minecraft:textures/atlas/beds.png-atlas
[11:12:28] [Render thread/INFO]: Created: 1024x512x0 minecraft:textures/atlas/items.png-atlas
[11:12:28] [Render thread/INFO]: Created: 1024x1024x0 minecraft:textures/atlas/gui.png-atlas
[11:12:28] [Render thread/INFO]: Created: 128x64x0 minecraft:textures/atlas/map_decorations.png-atlas
[11:12:28] [Render thread/INFO]: Created: 512x256x0 minecraft:textures/atlas/signs.png-atlas
[11:12:28] [Render thread/INFO]: Created: 512x512x0 minecraft:textures/atlas/shulker_boxes.png-atlas
[11:12:32] [Render thread/INFO]: Connecting to mc.hypixel.net, 25565
[11:12:32] [Render thread/INFO]: Registering mod resource pack
[11:12:33] [Render thread/WARN]: Failed to send a packet as the client is not connected to a server 'ServerboundRegisterPacket{wantedPackets={hyevent:location=1}} VersionedPacket{version=1}'
[11:12:33] [Render thread/INFO]: [STDOUT]: AVAST! InterMethod[intermediary=method_3349, ownerType=InterClass[intermediary=Lnet/minecraft/class_776;], returnType=InterClass[intermediary=Lnet/minecraft/class_1087;], argumentTypes=[InterClass[intermediary=Lnet/minecraft/class_2680;]]]
[11:12:33] [Render thread/WARN]: Failed to send a packet as the client is not connected to a server 'VersionedPacket{version=2}'
[11:12:33] [Render thread/WARN]: Ambiguity between arguments [firmament, dev, reexportlore, all] and [firmament, dev, reexportlore, itemid] with inputs: [all]
[11:12:33] [Render thread/INFO]: [System] [CHAT] --------------  Guild: Message Of The Day  --------------
[11:12:33] [Render thread/INFO]: [System] [CHAT] | Flop.
[11:12:33] [Render thread/INFO]: [System] [CHAT] -----------------------------------------------------
[11:12:33] [Render thread/INFO]: [System] [CHAT]                          
[11:12:33] [Render thread/INFO]: Resizing Dynamic Transforms UBO, capacity limit of 2 reached during a single frame. New capacity will be 4.
[11:12:33] [Render thread/INFO]: Resizing Dynamic Transforms UBO, capacity limit of 4 reached during a single frame. New capacity will be 8.
[11:12:34] [Render thread/INFO]: Resizing Chunk Sections UBO, capacity limit of 2 reached during a single frame. New capacity will be 4.
[11:12:34] [Render thread/INFO]: Resizing Chunk Sections UBO, capacity limit of 4 reached during a single frame. New capacity will be 8.
[11:12:34] [Render thread/INFO]: Resizing Dynamic Transforms UBO, capacity limit of 8 reached during a single frame. New capacity will be 16.
[11:12:34] [Render thread/INFO]: [System] [CHAT] §b[MVP§2+§b] pointdock§f §6joined the lobby!
[11:12:34] [Render thread/INFO]: Resizing Chunk Sections UBO, capacity limit of 8 reached during a single frame. New capacity will be 32.
[11:12:34] [Render thread/INFO]: Resizing Chunk Sections UBO, capacity limit of 32 reached during a single frame. New capacity will be 64.
[11:12:34] [Render thread/INFO]: Resizing Dynamic Transforms UBO, capacity limit of 16 reached during a single frame. New capacity will be 32.
[11:12:34] [Render thread/INFO]: Resizing Chunk Sections UBO, capacity limit of 64 reached during a single frame. New capacity will be 128.
[11:12:35] [Render thread/INFO]: [System] [CHAT] §b[MVP§c+§b] beatbox4§f §6joined the lobby!
[11:12:36] [Render thread/INFO]: [System] [CHAT] §b[MVP§c+§b] MiniTheLazy§f §6joined the lobby!
[11:12:36] [Render thread/INFO]: [System] [CHAT]   ➤ You have reached your Hype limit! Add Hype to Prototype Lobby minigames by right-clicking with the Hype Diamond!
[11:12:40] [Render thread/INFO]: [System] [CHAT] Sending to server mini17U...
[11:12:40] [Render thread/WARN]: Ambiguity between arguments [firmament, dev, reexportlore, all] and [firmament, dev, reexportlore, itemid] with inputs: [all]
[11:12:40] [Render thread/INFO]: [System] [CHAT]        
[11:12:41] [Render thread/WARN]: Received an error response for packet hypixel:party_info: RATE_LIMITED
[11:12:41] [Render thread/WARN]: Received an error response for packet hypixel:party_info: RATE_LIMITED
[11:12:41] [Render thread/INFO]: [System] [CHAT] Since you've been away you earned 610,000 coins as interest in your personal bank account!
[11:12:42] [Render thread/INFO]: [System] [CHAT] §6§lGROSS! §7While you were offline, §2ൠ Pests §7spawned in §aPlots §b6§7, §b20§7, §b8§7, §b3§7 and §b13§7!
[11:12:42] [Render thread/INFO]: [System] [CHAT] Your ☘ Farming Fortune has been reduced by 75%!
[11:12:42] [Render thread/INFO]: [System] [CHAT] Click on a plot's name in chat to teleport to it!
[11:12:42] [Render thread/INFO]: [System] [CHAT] Firmament needs RoughlyEnoughItems to display its item list!\n - [Click here to download RoughlyEnoughItems]\n - [Click here to download Architectury API]\n - [Click here to download Cloth Config API]\nClick here to disable this warning
[11:12:43] [Render thread/INFO]: [System] [CHAT] 
[11:12:43] [Render thread/INFO]: [System] [CHAT] Welcome to Hypixel SkyBlock!
[11:12:43] [Render thread/INFO]: [System] [CHAT] BONUS! Temporarily earn 30% more skill experience!
[11:12:43] [Render thread/INFO]: [System] [CHAT] 
[11:12:43] [Render thread/INFO]: [System] [CHAT] §eLatest update: §bSkyBlock v0.24.2 §e§lCLICK
[11:12:44] [Render thread/INFO]: [System] [CHAT] §aYou are playing on profile: §eLemon
[11:12:44] [Render thread/INFO]: [System] [CHAT] Profile ID: dda9df5f-3c33-48ab-a2ed-9cd4a4ecad0d
[11:12:45] [Render thread/INFO]: [System] [CHAT] ALLOWANCE! You earned 50,000 coins!
[11:12:45] [Render thread/INFO]: [System] [CHAT] Warping...
[11:12:45] [Render thread/INFO]: [System] [CHAT] Sending to server mini17BD...
[11:12:45] [Render thread/WARN]: Ambiguity between arguments [firmament, dev, reexportlore, all] and [firmament, dev, reexportlore, itemid] with inputs: [all]
[11:12:46] [Render thread/INFO]: [System] [CHAT]        
[11:12:46] [Render thread/INFO]: Resizing Chunk Sections UBO, capacity limit of 128 reached during a single frame. New capacity will be 256.
[11:12:49] [Render thread/INFO]: [System] [CHAT] §aYou are playing on profile: §eLemon
[11:12:49] [Render thread/INFO]: [System] [CHAT] Profile ID: dda9df5f-3c33-48ab-a2ed-9cd4a4ecad0d
[11:12:51] [Render thread/INFO]: Resizing Dynamic Transforms UBO, capacity limit of 32 reached during a single frame. New capacity will be 64.
[11:12:51] [Render thread/INFO]: Resizing Dynamic Transforms UBO, capacity limit of 64 reached during a single frame. New capacity will be 128.
[11:12:59] [Render thread/INFO]: Resizing Chunk Sections UBO, capacity limit of 256 reached during a single frame. New capacity will be 512.
[11:13:02] [Render thread/WARN]: Requested creation of existing team 'fkt_Weakling '
[11:13:02] [Render thread/WARN]: Requested creation of existing team 'fkt_Weakling '
[11:13:03] [Render thread/WARN]: Requested creation of existing team 'fkt_Weakling '
[11:13:06] [Render thread/WARN]: Requested creation of existing team 'fkt_Weakling '
[11:13:08] [Render thread/INFO]: [System] [CHAT] ᠅ You've earned 8,500 Gemstone Powder from mining your first Gemstone of the day!
[11:13:13] [Render thread/INFO]: [System] [CHAT] PRISTINE! You found ❤ Flawed Ruby Gemstone x9!
[11:13:14] [Render thread/WARN]: Requested creation of existing team 'fkt_Weakling '
[11:13:14] [Render thread/INFO]: [System] [CHAT] PRISTINE! You found ❤ Flawed Ruby Gemstone x10!
[11:13:14] [Render thread/ERROR]: Unreported exception thrown!
java.lang.IllegalAccessError: Update to non-static final field moe.nea.firmament.gui.hud.MoulConfigHud$Companion$componentWrapper$2$1.field_22787 attempted from a different class (moe.nea.firmament.gui.hud.MoulConfigHud$Companion$componentWrapper$2$1) than the field's declaring class
	at knot//moe.nea.firmament.gui.hud.MoulConfigHud$Companion$componentWrapper$2$1.<init>(MoulConfigHud.kt:24)
	at knot//moe.nea.firmament.gui.hud.MoulConfigHud.componentWrapper_delegate$lambda$0(MoulConfigHud.kt:22)
	at knot//kotlin.SynchronizedLazyImpl.getValue(LazyJVM.kt:86)
	at knot//moe.nea.firmament.gui.hud.MoulConfigHud$Companion.getComponentWrapper(MoulConfigHud.kt:21)
	at knot//moe.nea.firmament.gui.hud.MoulConfigHud$Companion.access$getComponentWrapper(MoulConfigHud.kt:20)
	at knot//moe.nea.firmament.gui.hud.MoulConfigHud._init_$lambda$0(MoulConfigHud.kt:43)
	at knot//moe.nea.firmament.events.FirmamentEventBus.publish(FirmamentEventBus.kt:47)
	at knot//net.minecraft.class_329.handler$bcl000$firmament$renderCallBack(class_329.java:2535)
	at knot//net.minecraft.class_329.method_55799(class_329.java)
	at knot//net.minecraft.class_329.mixinextras$bridge$method_55799$153(class_329.java)
	at knot//net.minecraft.class_329.mdc55ee2$fabric-rendering-v1$lambda$wrapSleepOverlay$15$6(class_329.java:2142)
	at knot//net.fabricmc.fabric.impl.client.rendering.hud.HudElementRegistryImpl$RootLayer.render(HudElementRegistryImpl.java:216)
	at knot//net.minecraft.class_329.wrapOperation$znn000$fabric-rendering-v1$wrapSleepOverlay(class_329.java:2142)
	at knot//net.minecraft.class_329.method_1753(class_329.java:230)
	at knot//net.minecraft.class_757.method_3192(class_757.java:522)
	at knot//net.minecraft.class_310.method_1523(class_310.java:1393)
	at knot//net.minecraft.class_310.method_1514(class_310.java:966)
	at knot//net.minecraft.client.main.Main.main(Main.java:250)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:514)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:72)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:105)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70)
```

--- a/src/main/kotlin/gui/hud/MoulConfigHud.kt
+++ b/src/main/kotlin/gui/hud/MoulConfigHud.kt
@@ -19,11 +19,7 @@ abstract class MoulConfigHud(
 ) {
 	companion object {
 		private val componentWrapper by lazy {
-			object : MoulConfigScreenComponent(Component.empty(), GuiContext(TextComponent("§cERROR")), null) {
-				init {
-					this.minecraft = MC.instance
-				}
-			}
+			MoulConfigScreenComponent(Component.empty(), GuiContext(TextComponent("§cERROR")), null)
 		}
 	}
 

